CHANGES.rst

```diff
@@ -1,35 +1,37 @@
 Changes
 =======
 
-Version 0.9.0 (UNRELEASED)
----------------------------
+Version 0.9.0 (2022-12-14)
+--------------------------
 
-- Adds notification icon to the header to inform users when critical levels of quota usage is reached.
-- Adds a label link to the workflow launcher URL remote origin on the workflow details page.
 - Adds Launch on REANA page allowing the submission of workflows via badge-clicking.
+- Adds notifications to inform users when critical levels of quota usage is reached.
 - Adds 404 Not Found error page.
-- Adds periodic refresh of the workflow details page.
-- Adds new options to the workflow list page to sort workflows by most used disk and cpu quota.
 - Adds tab titles to all the pages.
-- Fixes redirection chain for non-signed-in CERN SSO users to access the desired target page after sign-in.
-- Fixes ``fetchWorkflow`` action to fetch a specific workflow instead of the entire user workflow list.
-- Fixes the ordering by size of the files showed in the ``Workspace`` tab of the workflow-details page.
 - Changes OAuth configuration to enable the new CERN SSO.
-- Changes the workflow-details page to refresh after the deletion of a workflow.
 - Changes the workflow-details page to show the logs of the workflow engine.
 - Changes the workflow-details page to show file sizes in a human-readable format.
 - Changes the workflow-details page to show the workspace's retention rules.
 - Changes the workflow-details page to show the duration of the workflow's jobs.
+- Changes the workflow-details page to display a label of the workflow launcher URL remote origin.
+- Changes the workflow-details page to periodically refresh the content of the page.
+- Changes the workflow-details page to refresh after the deletion of a workflow.
+- Changes the workflow-list page to add a way to hide deleted workflows.
+- Changes the workflow-list page to add new workflows sorting options by most used disk and cpu quota.
 - Changes the deletion of a workflow to always clean up the workspace.
 - Changes the announcements to support limited HTML markup.
+- Changes UI with styling improvements and some other small enhancements.
```

Comment:
Yes, it seems too generic in this way... If we make it more specific, then it might make more sense. For example "Changes styling of the launcher web page to be more responsive on narrow browser screens" or some such.

A good Litmus test is to ask whether the reader might think "... so what?" after each news item. I.e. the goal of each item should be nicely self-understood ("Changes ... so that ...")

Regarding covering all small change, we may want to revive the idea of automated generation of release news based on commit messages... Something for live musings.

Reply:
I've removed this item from the list